src/ivectorbin/ivector-compute-lda.cc

```diff
@@ -209,14 +221,17 @@ int main(int argc, char *argv[]) {
     ParseOptions po(usage);
 
     int32 lda_dim = 100; // Dimension we reduce to
-    BaseFloat total_covariance_factor = 0.0;
+    BaseFloat total_covariance_factor = 0.0,
+              covariance_floor = 1.0e-04;
     bool binary = true;
 
     po.Register("dim", &lda_dim, "Dimension we keep with the LDA transform");
     po.Register("total-covariance-factor", &total_covariance_factor,
                 "If this is 0.0 we normalize to make the within-class covariance "
                 "unit; if 1.0, the total covariance; if between, we normalize "
                 "an interpolated matrix.");
+    po.Register("covariance-floor", &covariance_floor, "Floor on the eigenvalues "
+                " of the interpolated covariance matrix");
```

Comment:
I'd add (relative to the highest eigenvalue)

Reply:
Now it's directly configurable, rather than being multiplied with the max eigenvalue. 

I was finding that setting the floor to max_eigenvalue * 1.0e-04 was changing my results slightly. That's why I was using a std::numeric_limits::min() in the first place--to get a smaller floor.